index.html

```diff
@@ -10,7 +10,7 @@
 		<meta property="og:description" content="Minecraft Double Smooth Stone Slab">
 		<meta property="og:type" content="website">
 		<meta property="og:image" content="https://minecraftdoublesmoothstoneslab.website/minecraftdoublesmoothstoneslab.jpg">
-		<link rel="stylesheet" type="text/css" href="/assets/style.css" />
+		<link rel="stylesheet" type="text/css" href="assets/style.css" />
```

Comment:
also revert this change
```suggestion
		<link rel="stylesheet" type="text/css" href="/assets/style.css" />
```